src/vignette/storage/static_assets.clj

```diff
@@ -24,7 +24,7 @@
 
 (defn create-async-response-stored-object [static-image-get image-review-get]
   (let [static-image-response @static-image-get]
-    (if (and (-> static-image-response :status (= 200)) (some #(= % (:status @image-review-get)) [200 404]))
+    (if (and (-> static-image-response :status (= 200)) (some #(= % (:status @image-review-get)) [200 404 400]))
```

Comment:
We need to concern about what happen when image-review will not work properly and it will throws 500.

With this implementation all images will be not display.